pyproject.toml

```diff
@@ -11,35 +11,36 @@ license = "MIT"
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.8"
-click = ">=8.1.6"
-curies = ">=0.7.3"
-linkml-runtime = ">=1.5.5"
-pandas = ">1.0.3"
-pansql = { version = ">=0.0.1", extras = ["pansql"] }
-sssom-schema = ">=0.14.0"
-networkx = { version = ">=3.1", extras = ["networkx"] }
-sparqlwrapper = ">=2.0.0"
-validators = ">=0.20.0"
+python = "^3.9"
+click = ">=8.1.7"
+curies = ">=0.7.8"
+pandas = ">=2.2.1"
+linkml-runtime = "^1.7.5"
+sssom-schema = "^0.15.2"
+SPARQLWrapper = ">=2.0.0"
+validators = ">=0.25.0"
 deprecation = "^2.1.0"
-pyyaml = "^6.0.1"
-rdflib = ">=6.0.0"
+PyYAML = "^6.0.1"
+rdflib = ">=7.0.0"
```

Comment:
Pretty steep. Necessary?

Reply:
might as well if it isn't breaking for us.